### ISSUE TITLE
[Fixed in GH-6052] Safari: Missing Visual Focus on Checkout: Add New Payment

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4325]
![Screen Shot 2019-08-29 at 7 14 53 PM](https://user-images.githubusercontent.com/15113219/63982932-85c7a380-ca92-11e9-8817-947311257c73.png)

**Comment by KateChuen** [comment]
Tested on dev3. I'm still seeing the missing focus on:
- Account Holder Name input field
- Security Code input field
- Set as default checkbox

- The focus on the buttons are grey instead of blue.

**Comment by developpeurweb** [comment]
**For QA:**

_The focus on the buttons are grey instead of blue._
The grey outline comes from Bootstrap, this is fine for now, it should go away when we remove that framework.

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 1 13 41 PM](https://user-images.githubusercontent.com/15113219/73563714-087ed100-442c-11ea-9123-42a0e4320645.png)
![Screen Shot 2020-01-31 at 1 13 53 PM](https://user-images.githubusercontent.com/15113219/73563719-0b79c180-442c-11ea-82f8-3a388e727d56.png)
![Screen Shot 2020-01-31 at 1 14 03 PM](https://user-images.githubusercontent.com/15113219/73563725-0e74b200-442c-11ea-9256-06ff5bdbeec7.png)
![Screen Shot 2020-01-31 at 1 14 25 PM](https://user-images.githubusercontent.com/15113219/73563728-10d70c00-442c-11ea-8f59-541cfab02266.png)

**Comment by KateChuen** [comment]
Tested on dev3
- Chrome ✅
- Firefox ✅
- Safari ✅
